docs/user/nifi/examples.rst

```diff
@@ -0,0 +1,55 @@
+Examples
+--------
+
+Ingest a GeoMesa Kafka Topic into HBase
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A common use case is to store the most recent state of an object in a GeoMesa Kafka data store to drive a live map
+view, but to persist every state of the object into long-term storage in HBase or Accumulo for history and
+analytics. This example will show how to pull data off a GeoMesa Kafka topic, and persist it in HBase with
+minimal configuration.
+
+Install the GeoMesa NARs
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow the steps under :ref:`nifi_install` to install the GeoMesa processors in your NiFi instance. This
+tutorial requires the GeoMesa Kafka and HBase NARs, as well as the standard GeoMesa service NARs.
+
+Add and Configure the Processors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to add two processors to your flow, a ``GetGeoMesaKafkaRecord`` processor and an
+``AvroToPutGeoMesaHBase`` processor. Connect the output of the ``Get`` processor to the input of the ``Put``
+processor, and auto-terminate all the other connections:
+
+.. image:: /user/_static/img/nifi-kafka-to-hbase-flow.png
+   :align: center
+
+Next, create a new controller service by clicking 'Configure' on the NiFi flow, then going to the Controller
+Services tab and clicking the ``+`` button, then selecting the ``GeoAvroRecordSetWriterFactory``. The
+``GeoAvroRecordSetWriterFactory`` can be left with its default configuration, which is to use NiFi expressions.
+The ``GeoGeoMesaKafkaRecord`` processor will populate the necessary attributes according to the feature type
+being read:
+
+.. image:: /user/_static/img/nifi-avro-record-writer-config.png
+   :align: center
+
+The ``GetGeoMesaKafkaRecord`` processor will pull data off of the GeoMesa Kafka topic and write it out using
+a NiFi record writer. The processor can output data in any format supported by the NiFi records API, but for
+this example we're going to use GeoAvro to simplify the ingest into HBase. Configure the processor by
+setting the appropriate Kafka connection parameters and setting the ``GeoAvroRecordWriterFactory`` you just
+created for the output. Set a unique Kafka group ID, to ensure that the processor reads all the data coming
```

Comment:
We require folks to set their own Kafka group ID?  

This means that folks duplicating a GetGeoMesaKafkaRecord processor could get surprised!

Reply:
well, the group id has to be the same for any given processor, but unique from all other processors. i don't know of a way to do that automatically...

Reply:
this may not be much of an issue either way - generally, you probably wouldn't duplicate the processor, you would just duplicate and split the output flow. if you are consuming a different topic, then it doesn't matter if the id is the same.